pandas/tests/io/formats/test_to_latex.py

```diff
@@ -61,15 +61,15 @@ def test_to_latex_to_file_utf8_without_encoding(self):
 
     def test_to_latex_tabular_with_index(self):
         df = DataFrame({"a": [1, 2], "b": ["b1", "b2"]})
-        result = df.to_latex()
+        result = df.to_latex(hrules=True)
```

Comment:
Is it OK, that in the previous implementations ``\toprule`` and the like were drawn by default, but now it is necessary to specify ``hrules=True`` explicitly?

Reply:
yeh I wondered about that as well, but the `Styler` default is to exclude them, and I do wonder if a better default is not to add extraneous items from an additional latex package requirement. (disregarding the current pandas implementation()